apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState, useEffect, useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import ModuleView from './ModuleView';
+
+import styles from './ModuleView.module.css';
```

Comment:
Since this is new code let's avoid using CSS modules and use inline tailwind classes instead.